.vscode/extensions.json

```
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    // We use Markdown All in One to generate the table of contents.
```

Use the same comment syntax as we do in Buzz About

```suggestion
    // Markdown All in One
    // https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
```